global-news-app/src/components/Button/Button.tsx

```diff
@@ -1,12 +1,14 @@
 import { ButtonProps } from "@mui/material";
 import { StyledButton } from "./styles";
 import ArrowRight from "assets/icons/ArrowButton";
-import { buttonVariants } from "assets/styles/types";
-
+import { ButtonVariants } from "Types/enums";
 
 export interface IButton extends ButtonProps {
   text: string;
-  buttontype: buttonVariants.PRIMARY | buttonVariants.SECONDARY | buttonVariants.TEXT;
+  buttontype:
```

Comment:
`buttontype: buttonVariants;`

you can directly specify the type of `buttonType` as the enum `buttonVariants` without listing each member